system/termcurses/tcurses_vt100.c

```diff
@@ -112,22 +112,22 @@ static const struct termcurses_ops_s g_vt100_ops =
 
 /* VT100 terminal codes */
 
-static const char *g_clrscr         = "\033[2J";      /* Clear screen */
-static const char *g_clreos         = "\033[J";       /* Clear to end of screen */
-static const char *g_clreol         = "\033[K";       /* Clear to end of line */
-
-static const char *g_movecurs       = "\033[%d;%dH";  /* Move cursor to x,y */
-static const char *g_getwinsize     = "\x1b[s\x1b[999;999H\x1b[6n\x1bu";
-static const char *g_setfgcolor     = "\x1b[38;5;%dm";
```

Comment:
it's more simple to change %d to %c?

Reply:
Yes, you are right. @xiaoxiang781216. This happens when you try to fix something at very late hours. :) I must have hesitated signed char formatting with an unsigned char value. 

However, we still have a problem with the sequence. On minicom, the 'm' character gets printed out and becomes visible.